src/components/AddPlaidBankAccount.js

```diff
@@ -82,9 +87,19 @@ function AddPlaidBankAccount({
     bankAccountID,
     allowDebit,
     isPlaidDisabled,
+    isDisplayedInNewVBBA,
 }) {
+    const plaidBankAccounts = lodashGet(plaidData, 'bankAccounts') || [];
```

Comment:
shouldn't the default be within lodashGet?

Reply:
yes, Ive moved this line and didnt pay that much attention to it. Fixing it